components/Form/index.js

```diff
@@ -116,12 +183,69 @@ export default function Form({
             <label htmlFor="location">Location:</label>
             <input type="text" id="location" name="location" />
             <label htmlFor="tasks">Tasks:</label>
-            <textarea type="text" id="tasks" name="tasks" />
+            <textarea type="text" maxLength={300} id="tasks" name="tasks" />
             <label htmlFor="ideas">Ideas, Message, Thoughts:</label>
-            <textarea type="text" id="ideas" name="ideas" />
+            <textarea type="text" maxLength={400} id="ideas" name="ideas" />
             <label htmlFor="guests">Guests:</label>
-            <textarea type="text" id="guests" name="guests" />
-            <Button type="submit">Save</Button>
+            <textarea type="text" maxLength={100} id="guests" name="guests" />
+            <fieldset>
+              <legend>EXPENSES</legend>
+              <label htmlFor="budget">Budget</label>
+              <input
+                type="number"
+                min="0.00"
+                max="10000.00"
+                step="0.01"
+                id="budget"
```

Comment:
There is a lot of repetition going on here which does not fit to the DRY rule. I suggest to figure out the differences between the forms (`isEditing`) and only change the lines depending on this like the `defaultValue` attribute of your input fields and the kind of the save / update button.